AntiCovid/app/src/androidTest/java/com/swtug/anticovid/startUp/RegisterTest.kt

```diff
@@ -45,7 +44,7 @@ class RegisterTest {
         }
     }
 
-    private fun fillOutRegisterForm(email: String, pwd: String, secID: String) {
+    private fun fillOutRegisterForm(email: String, pwd: String) {
```

Comment:
Please revert these changes. Since you actually provided a newer version already for the whole test